Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/GmailMailHelperTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using NUnit.Framework;
+using Relativity.Test.Helpers.Mail;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+
+namespace Relativity.Test.Helpers.NUnit.Integration.Mail
+{
+	public class GmailMailHelperTests
+	{
+		private IMailHelper SuT;
+
+		/// <summary>
+		/// You will need to have a Gmail account setup with API permissions and oAuth
+		/// 
+		/// Also this test assumes you have an email already in the MailTrap Inbox
+		/// </summary>
+		private const string ApiKey = "";
+		private const string EmailTestSubject = "Relativity Integration Test";
+		private const string EmailTestBody = "Relativity test email for integration tests";
+		private const string EmailTestDisplayName = "Relativity ODA";
+		private const int EmailPort = 587;
+		private const string EmailDomain = "smtp.gmail.com";
+
+		private const string EmailAddress = "";
+		private const string EmailPassword = "";
+
+		[OneTimeSetUp]
+		public void SetUp()
+		{
+			SuT = new GmailMailHelper(EmailAddress, EmailPassword);
+		}
+
+		[OneTimeTearDown]
+		public void TearDown()
+		{
+			SuT = null;
+		}
+
+		[Test]
+		public void GetInboxes()
+		{
+			// Arrange
+
+			// Act 
+			List<IMailInboxModel> inboxes = SuT.GetInboxes();
+
+			// Assert
+			Assert.Greater(inboxes.Count, 0);
+			Assert.IsFalse(string.IsNullOrEmpty(inboxes.First().Id));
+		}
+
+		[Test]
+		public void GetMessages()
+		{
+			// Arrange
+			List<IMailInboxModel> inboxes = SuT.GetInboxes();
+
+			SendMail();
+
+			// Act 
+			List<IMailMessageModel> messages = SuT.GetMessagesInInbox(inboxes.First());
+
+			// Assert
+			Assert.Greater(messages.Count, 0);
+			Assert.IsFalse(string.IsNullOrEmpty(messages.First().Id));
+
+			IMailMessageModel message = SuT.DeleteMessage(inboxes.First(), messages.First().Id);

```

Comment:
you should also Assert inboxes.First() and messages.First() are not null before this assert.